client/client.js

```diff
@@ -405,3 +405,20 @@ function expressMetricsEndpointDetail(modelOrUri, windowStartTime, callback) {
   );
 }
 Client.prototype.expressMetricsEndpointDetail = expressMetricsEndpointDetail;
+
+function executorCreate(driver, callback) {
```

Comment:
This is almost certainly going to be expanded, perhaps it should start off as an object so it doesn't need to be changed later?